snuba/cli/migrations.py

```diff
@@ -52,6 +55,12 @@ def list() -> None:
 
 @migrations.command()
 @click.option("-g", "--group", default=None)
+@click.option(
+    "-r",
+    "--readiness_state",
```

Comment:
nit: i would make this `--readiness-state` since we use `-` over `_` in other options